src/basic/process-util.h

```diff
@@ -210,3 +210,8 @@ int get_process_threads(pid_t pid);
 
 int is_reaper_process(void);
 int make_reaper_process(bool b);
+
+/* Forks and invokes 'path' with 'argv' and 'envp' using CLONE_VM and CLONE_VFORK, which means the caller will
+ * be blocked until the child either exits or exec's. The memory of the child will be fully shared with the
+ * memory of the parent, so that there are no copy-on-write or memory.max issues. */
```

Comment:
I'd really put that in the .c file. that's where I look at least if i try to grok what a function does. I don#t really care so much about the header file then